packages/menu/src/menu-group.css

```
@@ -19,6 +19,35 @@ governing permissions and limitations under the License.
     flex-direction: column;
 }
 
+/**
+ * The following corrects for positioning of the header when changing the
+ * direction of the content while https://github.com/adobe/spectrum-css/issues/794
+ * persists at the Spectrum CSS level.
+ */
+:host([dir='ltr']) .header {
+    /* .spectrum-Menu-sectionHeading */
+    padding: 0 var(--spectrum-global-dimension-size-450) 0
+        var(--spectrum-global-dimension-size-150);
+}
+
+:host([dir='rtl']) .header {
+    /* .spectrum-Menu-sectionHeading */
+    padding: 0 var(--spectrum-global-dimension-size-150) 0
+        var(--spectrum-global-dimension-size-450);
+}
+
+sp-menu {
+    --swc-menu-width: 100%;
+}
+
+:host(:last-child) sp-menu {
+    margin-bottom: 0;
+}
+
+:host(:first-child) .header[hidden] + sp-menu {
+    margin-top: 0;
+}
+
```

I've learned from experience that it's much harder to remove a workaround if you don't mark where it ends
```suggestion
/**
 * End workaround for https://github.com/adobe/spectrum-css/issues/794
 */
```